test/parallel/test-stringbytes-external.js

```diff
@@ -119,3 +119,68 @@ var PRE_3OF4_APEX = Math.ceil((EXTERN_APEX / 4) * 3) - RADIOS;
   assert.equal(a, b);
   assert.equal(b, c);
 })();
+
+// v8 fails silently if string length > v8::String::kMaxLength
+(function() {
+  // v8::String::kMaxLength defined in v8.h
+  const kStringMaxLength = process.binding('buffer').kStringMaxLength;
+
+  assert.throws(function() {
+    var buf = new Buffer(kStringMaxLength + 1).fill().toString();
```

Comment:
Remove the `fill()`. `kMaxStringLength` is pretty large and will add unnecessary time to the test. Just use `toString('binary')` to ensure the correct string length is achieved.

/cc @bnoordhuis Some issue I might be missing by not filling it?